src/abstract_visualizer.jl

```diff
@@ -8,6 +8,7 @@ function setprop! end
 function setcontrol! end
 
 
-# Convenient shortcuts for creating new objects
+# Convenient shortcuts
 setobject!(vis::AbstractVisualizer, geom::GeometryLike) = setobject!(vis, Object(geom))
 setobject!(vis::AbstractVisualizer, geom::GeometryLike, material::AbstractMaterial) = setobject!(vis, Object(geom, material))
+hide!(vis::AbstractVisualizer) = settransform!(vis, LinearMap(UniformScaling(false)))
```

Comment:
I think `setprop!(vis, "visible", false)` might be a more idiomatic way to do this. What do you think?

Reply:
Yes. I thought you might have a suggestion like that, but I didn't know the right API call. I'll change it.